server/collaborator/src/platform.ts

```diff
@@ -18,7 +18,7 @@ import { createClient, getTransactorEndpoint } from '@hcengineering/server-clien
 import { Token, generateToken } from '@hcengineering/server-token'
 
 async function connect (token: string): Promise<Client> {
-  const endpoint = await getTransactorEndpoint(token)
+  const endpoint = await getTransactorEndpoint(token, 'external')
```

Comment:
Why? collaborator should use internal URL for transactor endpoint.

Reply:
was used for debugging, removed